Microsoft.Toolkit.Uwp.SampleApp/App.xaml

```diff
@@ -6,6 +6,7 @@
 
     <Application.Resources>
         <ResourceDictionary>
+            <!-- code comment -->
```

Comment:
?

Reply:
!

Reply:
I faced a sluggish CI Build, and was hoping to kickstart a new build by making a small change.  Comment removed.